pyqt5-gui/about.py

```diff
@@ -0,0 +1,84 @@
+import sys
+
+from PyQt5.QtWidgets import (
+    QVBoxLayout,
+    QHBoxLayout,
+    QPushButton,
+    QLabel,
+    QWidget,
+    QApplication
+)
+from PyQt5.QtGui import *
+from PyQt5.QtCore import *
+
+
+class About(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.setWindowTitle("About Us")
+        self.setGeometry(650, 300, 600, 400)
+
```

Comment:
you the `.setWindowIcon()` method to set the window icon. You can use the file `market.png` in img folder